src/core/tsi/ssl_transport_security.cc

```diff
@@ -1146,6 +1211,17 @@ static tsi_result ssl_handshaker_result_extract_peer(
       &peer->properties[peer->property_count]);
   if (result != TSI_OK) return result;
   peer->property_count++;
+
+  if (ca_cert != nullptr) {
+    result = peer_property_from_x509_subject(
+        ca_cert, &peer->properties[peer->property_count], true);
+    if (result != TSI_OK) {
+      gpr_log(GPR_ERROR, "resut: %d", static_cast<int>(result));
+      return result;
```

Comment:
We shouldn't fail if there is a problem here, I think we should just log and continue.

Reply:
Ack, will change